fluentui_core/src/main/java/com/microsoft/fluentui/compose/ModalPopup.kt

```diff
@@ -93,6 +104,20 @@ fun ModalPopup(
         }
     }
 }
+@Composable
+fun convertWindowInsetsCompatTypeToWindowInsets(windowInsetsCompatType: Int): WindowInsets {
```

Comment:
Please explore creating WindowInsetsCompat & getting WindowInset out of it to avoid this function.